plugins/com.python.pydev.analysis/src/com/python/pydev/analysis/pylint/PyLintPreferences.java

```diff
@@ -104,6 +108,10 @@ public static int rSeverity() {
         return PydevPrefs.getEclipsePreferences().getInt(SEVERITY_REFACTOR, DEFAULT_SEVERITY_REFACTOR);
     }
 
+    public static int iSeverity() {
```

Comment:
I think you meant `infoSeverity()`.

Reply:
I just kept and followed the pattern of the previous similar statements

i.e: 
- wSeverity
- eSeverity
.
.
.
- rSeverity

Reply:
Ok, seems fair (I hadn't seen the others).